### DISCUSSION TITLE
Google Map Disappear after validation

### DISCUSSION THREAD

**Discussion by lovecoding-git** [livewire/livewire#4839] [1. Help]
I'm rendering google map using @push("script") in blade.
Map rendering part is in livewire component.
When I submit a form and the validation is failed, the map part is disappeared.

I think javascript is rendered once.
Is there any solution to keep the map (including changes) in livewire?
Thank you!

**Comment by joshhanley** [comment]
@lovecoding-git if you are using a javascript library that modifies the DOM, you need to make use of `wire:ignore` to ensure Livewire doesn't replace it. See https://laravel-livewire.com/docs/2.x/alpine-js#ignoring-dom-changes

Hope this helps!
